src/components/content/content-blocks/code-block/code-block.st.css

```
@@ -0,0 +1,5 @@
+@namespace "CodeBlock";
+
+.root {
+	white-space: pre;
```

I tested the code block, and the styling has a bug - it cuts long lines.
```suggestion
	white-space: pre-wrap;
```
seems to solve this. Also, consider adding browser support for this:
```suggestion
	white-space: pre-wrap;       /* Since CSS 2.1 */
	white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
	white-space: -pre-wrap;      /* Opera 4-6 */
	white-space: -o-pre-wrap;    /* Opera 7 */
```

Sources:
- [StackOverflow](https://www.longren.io/wrapping-text-inside-pre-tags/)
- [Wrapping Text Inside Pre Tags](https://www.longren.io/wrapping-text-inside-pre-tags/)